knowledge-base/how-to-display-a-report-via-instancereportsource-in-the-html5-viewer.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to display a report via InstanceReportSource in the HTML5 Viewer
+description: How to display a report via InstanceReportSource in the HTML5 Viewer.
```

Comment:
title, description and page_title should differ. The idea is that using different words to state the problem helps in SEO. In other words if you manage to add more keywords, the article will be more findable from Google search, thus more people will land on it. This is better described here - https://wiki.progress.com/display/DTS/Customer+Context and the whole KB content standard can be found here - https://wiki.progress.com/display/DTS/Knowledge+Base+Content+Standards#KnowledgeBaseContentStandards-Metadata. Given that, you can change the description to something like: Using InstanceReportSource to display reports in the HTML5 Viewer.